lantern_hnsw/test/expected/hnsw_vector.out

```diff
@@ -199,48 +184,43 @@ CREATE INDEX l2_idx ON small_world_arr USING lantern_hnsw(v) WITH (dim=3, m=2);
 INFO:  done init usearch index
 INFO:  inserted 3 elements
 INFO:  done saving 3 vectors
-EXPLAIN (COSTS FALSE) SELECT id FROM small_world_arr ORDER BY v <?> ARRAY[0,0,0];
+EXPLAIN (COSTS FALSE) SELECT id FROM small_world_arr ORDER BY v <-> ARRAY[0,0,0];
                  QUERY PLAN                 
 --------------------------------------------
  Index Scan using l2_idx on small_world_arr
-   Order By: (v <?> '{0,0,0}'::real[])
-(2 rows)
+   Order By: (v <-> '{0,0,0}'::real[])
 
-SELECT id FROM small_world_arr ORDER BY v <?> ARRAY[0,0,0];
+SELECT id FROM small_world_arr ORDER BY v <-> ARRAY[0,0,0];
  id 
 ----
   1
   2
   3
-(3 rows)
 
 DROP INDEX l2_idx;
 CREATE INDEX cos_idx ON small_world_arr USING lantern_hnsw(v) WITH (m=2);
 INFO:  done init usearch index
 INFO:  inserted 3 elements
 INFO:  done saving 3 vectors
-SELECT id FROM small_world_arr ORDER BY v <?> ARRAY[0,0,0];
+SELECT id FROM small_world_arr ORDER BY v <=> ARRAY[0,0,0];
  id 
 ----
   1
   2
   3
-(3 rows)
 
 DROP INDEX cos_idx;
 CREATE INDEX ham_idx ON small_world_arr USING lantern_hnsw(v) WITH (m=3);
```

Comment:
was this supposed to be testing hamming distance? it seems the index creation uses the default l2sq operator in this case as well - there is a typo in the old test.

Reply:
fixed to create an index with correct opclass